src/truefoundry_sdk/types/agent_skill_source_blob_storage.py

```
@@ -0,0 +1,26 @@
+from ..core.pydantic_utilities import IS_PYDANTIC_V2, UniversalBaseModel
+
+
+class AgentSkillSourceBlobStorage(UniversalBaseModel):
+    """
+    SKILL.md content stored on blob
+    """
+
+    type: typing.Literal["blob-storage"] = "blob-storage"
+    description: str = pydantic.Field()
+    """
+    1–1024 chars.
```

Ruff (0.15.9)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)